src/ui/popup/regex.tsx

```diff
@@ -62,7 +62,15 @@ export default function Regex(props: {
 	tab: Resource<ManagerTab>;
 }) {
 	return (
-		<div class={styles.regexContainer}>
+		<div
+			class={styles.regexContainer}
+			onKeyDown={(event) => {
+				console.log(event);
+				if (event.key === 'Enter') {
```

Comment:
```suggestion
				if (event.key === 'Enter' && !event.isComposing) {
```

Reply:
This causes errors with some IMEs for languages like chinese and japanese

Reply:
I didn't even know the existence of this property 😅. Going to commit the update in a moment.

Reply:
changes made here -> https://github.com/web-scrobbler/web-scrobbler/pull/3785/commits/d5d993d8a34833f897bbcf2a61196ee5cd363ab1